code/modules/vending/cartridge.dm

```diff
@@ -12,11 +12,11 @@
 					/obj/item/computer_hardware/hard_drive/role/janitor = 10,
 					/obj/item/computer_hardware/hard_drive/role/signal/toxins = 10,
 					/obj/item/modular_computer/tablet/pda/heads = 10,
-					/obj/item/computer_hardware/hard_drive/role/captain = 3,
-					/obj/item/computer_hardware/hard_drive/role/cargo_technician = 10)
-	premium = list(/obj/item/computer_hardware/hard_drive/role/unlicensed = 3)
+					/obj/item/computer_hardware/hard_drive/role/cargo_technician = 10,
+					/obj/item/computer_hardware/hard_drive/role/unlicensed = 3)
```

Comment:
these are only supposed to be obtainable by hacking

Reply:
I don't think it's supposed to be obtainable by hacking. `/obj/item/computer_hardware/hard_drive/role/unlicensed` was previously `/cartridge/annoyance/lesser` which is lesser version of VIP spam cartridge, and it was added as an easy for the spam feature.

Reply:
![image](https://user-images.githubusercontent.com/87972842/196153564-3dfcc97d-89b2-4120-a202-1cf3d018587f.png)

Reply:
it is now `/obj/item/computer_hardware/hard_drive/role/maint`, and I am going to make a PR in the future that it can spawn as random maint item, so it's renamed as maint.